proc/process.c

```diff
@@ -1369,6 +1369,10 @@ static void process_vforkThread(void *arg)
 	hal_memcpy(&current->process->tls, &parent->process->tls, sizeof(hal_tls_t));
 	hal_memcpy(&current->tls, &parent->tls, sizeof(hal_tls_t));
 
+	if (proc_resourcesCopy(parent->process) < 0) {
+		proc_threadEnd();
```

Comment:
Ah, sorry, missed that. We need to wakeup the parent here (like in `if` in 1355)

Reply:
And free parentkstack